src/osd/ECBackend.cc

```diff
@@ -992,7 +992,7 @@ void ECBackend::handle_sub_read(
       hinfo = get_hash_info(i->first);
       if (!hinfo) {
 	r = -EIO;
-	get_parent()->clog_error() << __func__ << ": No hinfo for " << i->first;
+	get_parent()->clog_error() << "No hinfo for " << i->first;
```

Comment:
hinfo doesn't mean much to a user - perhaps something like "Corruption detected: object i->first is missing hash_info"